Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -78,14 +78,11 @@ def fix_dynamic_linkage
   def expand_rpath(file, bad_name)
     suffix = bad_name.sub(/^@rpath/, "")
 
-    # short circuit: we expect lib to be usually correct, so we try it first
-    return (lib / suffix) if (lib / suffix).exist?
-
     file.rpaths.each do |rpath|
-      return (rpath / suffix) if (rpath / suffix).exist?
+      return (rpath/suffix) if (rpath/suffix).exist?
```

Comment:
I prefer no parentheses around `return (rpath/suffix)` myself. They're not necessary.